docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
+
+Select the filter plugin to use from the list of plugins given. If a filter you need has not been installed you may install it by clicking on the *Install from available plugins* link below the list.
+
+Give the filter a name, names must be unique within the Fledge instance. Once complete click on the *Next* button to configure your chosen filter plugin.
+
+Here we have assumed you selected the *RMS* plugin and see the configuration screen for that plugin.
+
++---------------------+
+| |flow_filterconfig| | 
++---------------------+
+
+Configure your plugin and then click on *Done*. The filter screen will be displayed with the empty filter now replaced with your newly defined and configured filter.
+
++-------------------+
+| |flow_filterdone| |
++-------------------+
+
+.. note::
+
+   The colour of the filter components outline is used to distinguish the type of element it is. South plugins are shown in green and filters in yellow. Filters also display a "funnel" icon in the centre of the element.
+
+Similar controls are shown around the periphery of the filter icon to those shown on the south plugin. There is no count display when you hover over the filter and no readings or logs are available that related just to this filter.
+
+The context menu has a single entry, *Delete*. This will delete the filter from the pipeline.
+
+The filter element also has two green dots that represent the ingress and egress points of the filter.
+
+Reordering Filters
+~~~~~~~~~~~~~~~~~~
+
+Using the flow based pipeline editor the process of reordering filters within a pipeline is simply a case of clicking and dragging the filter you wish to move it.
+
++-----------------+
+| |flow_dragging| |
++-----------------+
+
+In this case the filter called *rename* has been dragged from its position between *Sinusoid* and *SineRMS* and will be dropped on the connection coming from *SinRMS* to the storage layer.
+
+Releasing the mouse button to drop the filter on the connection will cause the pipeline to be drawn with the new filter order.
+
++------------------+
+| |flow_reordered| |
++------------------+
+
+Accessing An Existing Pipeline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To access an existing pipeline from the *South* or *North* screen you may either click on the service name or select the *Details* item from the services context menu.
+
++----------------+
+| |flow_details| |
++----------------+
+
+Canvas Action Bar
+~~~~~~~~~~~~~~~~~
+
+At the top of the working canvas for the pipeline flow editor a small action bar allows for some actions related to the drawing of the pipeline.
+
++------------------+
+| |flow_actionbar| |
++------------------+
+
+From left to right in the action bar the operations support are:
+
+  - **Reload** - reloads the pipeline, discarding any part complete actions.
+
+  - **Reset** - Resets the canvas to include the pipeline, useful if you have scrolled or zoomed the display and want to revert to seeing the entire pipeline.
```

Comment:
to include the pipeline, useful if you have -> to include the pipeline. This is useful if you have